<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Service</class>
 <widget class="QWidget" name="Service">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>323</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 2px;
border-color: #20B2AA;
border-radius: 3px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 2px;
border-color: #20B2AA;
border-radius: 3px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>43</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(32, 178, 170);
border-style: solid;
border-width: 2px;
border-color: #20B2AA;
border-radius: 3px;</string>
   </property>
   <property name="text">
    <string>전송</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
